Platform/OpenCanopy/BitmapFont.c

```diff
@@ -84,6 +84,9 @@ BmfGetKerningPair (
   ASSERT (Context != NULL);
 
   Pairs = Context->KerningPairs;
+
+  if (Pairs == NULL)
+    return NULL;
```

Comment:
Should this be in curly braces?

Reply:
Well, I showed you the decoder function from Apple firmware. There is no clut there, they just map 16 values to opacity levels with a step of 0x11. I believe we should follow.